src/engine/qcommon/qcommon.h

```diff
@@ -285,7 +288,7 @@ You or the server may be running older versions of the game."
 
 #define PROTOCOL_VERSION       86
 
-#define URI_SCHEME             GAMENAME_STRING "://"
+#define URI_SCHEME             "dgp://"
```

Comment:
By the way, it's not engine's task to redirect url calls to engine, not a problem to have both unvanquished and xonotic reads the url scheme if the desktop knows how to redirect these url to xonotic or unvanquished (I don't have magical solution, though). The problem of this kind of url stuff is it's more the job for a launcher (like the unvanquished updater), or a job for a game library manager like Steam or Lutris. If we want a common url for multiple games, it's something to design outside of Unvanquished : it's designing a standard with protocols, formats and common tools others can reuse, praising for adoption. :neutral_face: